crypto/types/multisig/multisignature.go

```diff
@@ -68,11 +78,10 @@ func (mSig *Multisignature) AddSignatureFromPubKey(sig []byte, pubkey crypto.Pub
 		return fmt.Errorf("provided key %X doesn't exist in pubkeys: \n%s", pubkey.Bytes(), strings.Join(keysStr, "\n"))
 	}
 
-	mSig.AddSignature(sig, index)
+	AddSignature(mSig, sig, index)
 	return nil
 }
 
-// Marshal the multisignature with amino
-func (mSig *Multisignature) Marshal() []byte {
-	return cdc.MustMarshalBinaryBare(mSig)
+func AddSignatureV2(mSig *signing.MultiSignatureData, sig signing.SignatureV2, keys []crypto.PubKey) error {
```

Comment:
can we write an integration test that:

1. uses `StdSignatureToSignatureV2` to get a signatureV2
2. Use the sig from 1 to add it to a multisig with `AddSignatureV2`

Reply:
Yes, added a test for the same